src/ELZ-0-2-0-beta/Test/Serving-XIS/elz-serving-xis-scenario-1-maximal.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://hl7.org/fhir/STU3/testscript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TestScript xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript"
+    nts:scenario="server">
+    <id value="elz-serving-xis-scenario-1-maximal"/>
+    <version value="R4"/>
+    <name value="ELZ - Serving XIS - Scenario 1 - Serving ELZ resources of van-XXX-Sanderink-de-Kok"/>
+    <description value="Scenario 1 - Serving ELZ resources of van-XXX-Sanderink-de-Kok"/>
+    <nts:authToken patientResourceId="patient-van-XXX-Sanderink-de-Kok"/>
+
+    <test id="01-serve-Task">
+        <name value="Scenario 1.1 - Task"/>
+        <description
+            value="Serve Task resource"/>
+        <nts:include value="test.server.successfulRead" scope="common"
+            resource="Task"
+            params="?_id=task-hg-VerwijzingHuisartsParamedicusMaximaalbericht"
+            useToken="true"/>
```

Comment:
Als het gaat om: wat wordt hier nou daadwerkelijk getest. Het valt me op dat het een vrij kort rijtje is met asserts - in vergelijking met de successfulSearch-variant. Klopt dat wel?

Sowieso mist misschien wel de belangrijkste controle: de validatie.

Ik _denk_ dat dat gewoon komt omdat er tot nu toe weinig tests worden met Reads, dus die common-compnonents bestaan nog niet of zijn nog niet volwassen genoeg. Ik denk aan jullie de schone taak om dat nu alsnog voor elkaar te gaan krijgen lijkt me